README.md

```diff
@@ -10,7 +10,52 @@ None
 
 | variable | description | default |
 |----------|-------------|---------|
+| `virtualbox_user` | Virtualbox user | `{{ __virtualbox_user }}` |
+| `virtualbox_group` | Virtualbox group | `{{ __virtualbox_group }}` |
+| `virtualbox_kernel_modules` | Virtualbox kernel modules | `{{ __virtualbox_kernel_modules|default('') }}` |
+| `virtualbox_version_redhat` | Virtualbox version in RedHat | `{{ __virtualbox_version_redhat|default('') }}` |
```

Comment:
found out why; the repo has multiple vagrant packages. would you mention the URL where users can find available versions?

http://download.virtualbox.org/virtualbox/rpm/rhel/7/